code/datums/ai/basic_mobs/targeting_strategies/basic_targeting_strategy.dm

```diff
@@ -50,7 +50,7 @@
 	if(living_mob.see_invisible < the_target.invisibility) //Target's invisible to us, forget it
 		return FALSE
 
-	if(isturf(living_mob.loc) && isturf(the_target.loc) && living_mob.z != the_target.z) // z check will always fail if target is in a mech or pawn is shapeshifted or jaunting
+	if(!ismob(living_mob.loc) && isturf(the_target.loc) && living_mob.z != the_target.z) // z check will always fail if target is in a mech or pawn is shapeshifted or jaunting
```

Comment:
This change makes no sense. This check is done to avoid ranged mobs from shooting at targets on a different z-level, it has nothing to do with the issue.

```suggestion
	if(!isturf(living_mob.loc))
		return FALSE
	if(isturf(the_target.loc) && living_mob.z != the_target.z) // z check will always fail if target is in a mech or pawn is shapeshifted or jaunting
```